src/pages/contact-us/index.tsx

```diff
@@ -1,95 +1,71 @@
 import React from 'react'
+import { useTranslation } from 'react-i18next'
+import ContactUsSendMessage from '@components/ContactUsSendMessage'
```

Comment:
make index.tsx file for this component

Reply:
Also, change component name because it sends not only a message

Reply:
done